### PR TITLE
Adds Asseblage Definition

### PR DESCRIPTION
**Pull request by joeyjoejoejr**

### CODE REVIEW

File: Glossary.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Jollyballs Glossary
 
 ## General Terms
+* **Asseblage**- A term used when calling a game to indicate that the players should gather at the table.
```
Comment (fatcatt316) [inline Glossary.md]: `playas` is preferred, but I don't feel strongly about this.

Comment (dneanover) [reply Glossary.md]: A glossary item should probably include a pronunciation.

Comment (fatcatt316) [reply Glossary.md]: /əˈseblɪdʒ/

**Comment by jpeterson-stats** [comment]
I'm rejecting this PR on the grounds that there's a typo. Surely, "Asseblage" should have been "Assemblage".

**Comment by joeyjoejoejr** [comment]
Somebody missed the joke

**Comment by fatcatt316** [comment]
This should probably be reopened and changed to a hotfix

**Pull request closed without merging**